docs/reference/objects/package/step.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: Package Step
+parent: Package
+---
+
+# Package Step
+{: .d-inline-block }
+object
+{: .label .fs-1 }
+
+Represents a creator-defined step along a [package]({% link docs/reference/objects/package.md %}) booking journey.
+
+<br>
+
+#### Attributes
+
+## `package_step.id`
+{: .d-inline-block }
+string
+{: .label .fs-1 }
+The database id of the step.
+
+## `package_step.name`
+{: .d-inline-block }
+string
+{: .label .fs-1 }
+The name given to the step.
+
+## `package_step.tagline`
+{: .d-inline-block }
+string
+{: .label .fs-1 }
+
+The description given to the step.
+
+## `package_step.product_groups`
+
+{: .d-inline-block }
+Array of packaged products
+{: .label .fs-1 }
+
+Each product containing variants added to the Package Step by the creator. Use `.variants` to get the variants added to the step for this product.
+
+## `package_step.required_start_date`
+
+{: .d-inline-block }
+Date
+{: .label .fs-1 }
+
+Creators may specify that bookers must choose to book accommodation within a specific date range. This will contain the start date of that range.
```

Comment:
The current wording suggests that the customer needs to book dates within a range rather than that the customers dates need to include the required dates, suggesting we reword slightly.
```suggestion
Creators may specify that customers must book accommodation dates which include a specific date range. This will return the start date of that range.
```